composer.json

```diff
@@ -8,20 +8,21 @@
         "ext-curl": "*",
         "ext-dom": "*",
         "ext-soap": "*",
+        "ext-xsl": "*",
         "cweagans/composer-patches": "^1",
-        "nyholm/psr7": "^1.4",
-        "php-http/guzzle6-adapter": "^2.0",
         "php-http/logger-plugin": "^1.3",
         "phpro/soap-client": "~2.0.0",
-        "psr/http-factory-implementation": "*",
-        "psr/log": "~1",
+        "psr/http-factory-implementation": "1.0",
+        "psr/log": "^1 || ^3",
         "symfony/serializer": "~4",
         "symfony/validator": "~4",
         "veewee/xml": "^1.0"
     },
     "require-dev": {
+        "php-http/guzzle6-adapter": "^2.0",
```

Comment:
Shouldn't we add this to the documentation, pointing out that the library users need a guzzle adapter?

Reply:
Oh I forgot something in that commit. I should have added this line: `        "psr/http-client-implementation": "^1",`

Basically, this line indicates to the user that this library requires a HTTP Client concrete implementation. If none are found, it will list a couple of libraries that are fullfilling that requirement and the user is able to choose whatever concrete implementations he wants to work with (Guzzle, Symfony, Slim, you name it... ).

We did the same in here: https://github.com/ecphp/cas-lib/blob/master/composer.json

Reply:
I just added it in https://github.com/openeuropa/epoetry-client/pull/51/commits/cd5875e19cdcdff915154ea46cf1c3a21887de18